build/aliases.yaml

```
@@ -23,9 +23,9 @@ test:
   - 'jshint'
 
 default:
+  - 'jshint'
```

Fail fast help us to fix faster